models/airlines.go

```diff
@@ -1,6 +1,7 @@
 package models
```

Comment:
shouldn't this file name also be changed to `airline.go`?

Reply:
yeah